JASP-Engine/JASP/R/regressionloglinearbayesian.R

```diff
@@ -15,811 +15,585 @@
+      container[["SubSummaryTable"]]$addFootnote(paste(round(logBlm.subestimates$post_prob, 3)), symbol = "<em>Posterior model probability =</em>")
 
-			if (length(bfObject$variables) > 0) {
+      if (length(bfObject$variables) > 0) {
+        
+        variablesInModel <- bfObject$variables
+        terms <- as.character(logBlm.subestimates$term)
+        coef  <- base::strsplit (terms, split = ":", fixed = TRUE)				
+        
+        for (var in seq_along(coef)) {
+          
+          terms      <- coef[[var]]
+          actualName <- list()
+          
+          for (j in seq_along(terms))
+            actualName[[j]] <- paste(lookup.table[[ terms[j] ]], collapse = " = ")
+          varName <- paste0(actualName, collapse = "*")
+          
+          results[[ len.Blogreg ]] <- list()
+          results[[ len.Blogreg ]]$"Name"      <- varName
+          post_mean <- as.numeric(logBlm.subestimates$post_mean[var])
+          results[[ len.Blogreg ]]$"post_mean" <- post_mean
+          post_var <- as.numeric(logBlm.subestimates$post_var[var])
+          results[[ len.Blogreg ]]$"post_var"  <- post_var
+          
+          if (options$regressionCoefficientsSubmodelCredibleIntervals){			
+            lower_lim <- as.numeric(logBlm.subestimates$lower[var])
+            results[[ len.Blogreg ]]$"lower_lim" <- lower_lim
+            upper_lim <- as.numeric(logBlm.subestimates$upper[var])
+            results[[ len.Blogreg ]]$"upper_lim" <- upper_lim
+          }
+          len.Blogreg <- len.Blogreg + 1
+        }		
+      }			
+      
+    } else {
+      
+      len.Blogreg <- length(results) + 1
+      if (length(bfObject$variables) > 0) {
+        
+        variablesInModel <- bfObject$variables
+        
+        len.Blogreg <- len.Blogreg + 1
+        
+        for (var in 1:length(variablesInModel)) {
+          
+          if (base::grepl(":", variablesInModel[var])) {
+            
+            # if interaction term					
+            vars <- unlist(strsplit(variablesInModel[var], split = ":"))
+            name <- paste0(vars, collapse = "\u2009\u273b\u2009")
+            
+          } else 
+            name <- as.character(variablesInModel[ var])
+          
+          results[[ len.Blogreg ]]$"Name" <- name
+          len.Blogreg <- len.Blogreg + 1
+        }
+      }
+    }
+    
+  } else {		
+    
+    len.Blogreg <- length(results) + 1
+    
+    if (length(bfObject$variables) > 0) 
+      variablesInModel <- bfObject$variables
+    
+    results[[ len.Blogreg ]]$"Model" <- 1
+  }
+  container[["SubSummaryTable"]]$addRows(results)
+}
 
-				variablesInModel <- bfObject$variables
-				
-			}
+# Container
+.basRegLogLinContainer <- function(jaspResults, dataset, options) {
+  if(is.null(jaspResults[["Container"]])) {
+    jaspResults[["Container"]] <- createJaspContainer()
+    jaspResults[["Container"]]$dependOn(c("counts", "modelTerms", 
+                                          "priorShape", "priorScale"))
```

Comment:
Shouldn't this also depend on the type of sampling (auto/manual) and the number of samples?